src/com/com_nrf24l01.cpp

```diff
@@ -0,0 +1,78 @@
+#include "com_nrf24l01.hpp"
+
+namespace com {
+
+auto NRF24L01::PutDataPacket(types::PutDataTarget target_id, types::ComDataPacket &packet) noexcept -> types::DriverStatus {
+  if (packet.data.size() > types::COM_MAX_DATA_FIELD_LENGTH) {
+    return types::DriverStatus::INPUT_ERROR;
+  }
+  //auto target_address = LookupComPartnerAddress(target_id);
+  data_pipe_address tmp_addr = {0xe7, 0xe7, 0xe7, 0xe7, 0xe7};
+  //ToDo: Wieder durch target_address ersetzen. Nur zu Testzwecken.
+  ON_ERROR_RETURN(nrf_->InitTx(tmp_addr));
+
+  auto payload = packet.Serialize();
+
+  ON_ERROR_RETURN(nrf_->SetTxPayload(payload));
+  ON_ERROR_RETURN(nrf_->GetIRQFlags().first);
+  ON_ERROR_RETURN(nrf_->SetChipEnable(State::ENABLED));
+  utilities::Sleep(1);
+  ON_ERROR_RETURN(nrf_->SetChipEnable(State::DISABLED));
+
+  for (int i = 0; i < 10; i++) {
```

Comment:
In the example code a waiting period of 1ms after transmission is suggested. There, a delay_us is implemented. The 10 iterations are a remnant of this construct. Do we need a delay_us as well?

Reply:
Maybe it would be clearer to me, if the 10 has a name that I can understand in this context. This is simply busy waiting for the get_irq_flg.second to have a specific state?

Here we would return types::DriverStatus::OK if we got the flag or not. How do we actually know if it worked. Or even more important. Is it important if it worked or not.

Reply:
Optimized tx sequence. Return value now depends on acutual status information of com device.